src/components/IterableEditComponent.tsx

```diff
@@ -241,7 +245,7 @@ export class IterableEditComponent extends React.Component<ImageUploadProps | It
                                 <div style={this.props.field.iterabletype.style ?
                                     Object.assign({}, this.props.field.iterabletype.style, { display: "inline-block" }) : { display: "inline-block" }}>
                                     <MultiSelectComponent
-                                        readonly={getReadOnly(this.props.field.iterabletype.readonly, currentModel) || this.props.readonly}
+                                        readonly={this.checkReadonly(this.props.field.iterabletype.readonly, currentModel)}
```

Comment:
this is repeated many times. pull it out.

Reply:
Updated